ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/config/SinkConfig.java

```diff
@@ -95,7 +96,8 @@ long getDefaultMaxOutstandingRequestBytes() {
             // BigQuery Streaming API Limits maximum rows per request to 10,000
             env.getInt(BATCH_MAX_MESSAGES, 10_000), // default 10K messages
             env.getDuration(BATCH_MAX_DELAY, "1s"), // default 1 second
-            env.getString(OUTPUT_TABLE), getFormat(env));
+            PubsubMessageToTemplatedString.forBigQuery(env.getString(OUTPUT_TABLE)),
```

Comment:
We change a few signatures to expect a `PubsubMessageToTemplatedString` rather than a bare String, so that we can control whether special BigQuery-specific transformations need to be applied. We provide two different static factory methods.